src/provider_manifests.py

```diff
@@ -199,3 +213,22 @@ def get_service_url(self, fqdn=False) -> Optional[str]:
         except Exception as e:
             log.error("Failed to get service url. ex=%s", e)
         return None
+
+    def apply_manifests(self):
+        """Apply the manifests to the cluster and restart deployments."""
+        super().apply_manifests()
+        for rsc in self.resources:
+            self.restart(rsc)
+
+    def restart(self, obj: HashableResource):
+        """Restart the hashable object if its possible to do so."""
+        timestamp = datetime.datetime.now(datetime.timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
What if we use a more standardized time format like RFC3339?
```suggestion
        timestamp = datetime.datetime.now(datetime.timezone.utc).isoformat()
```

Reply:
sure @mateoflorido no worries there